data/transactions/logic/assembler.go

```diff
@@ -909,43 +921,40 @@ func assembleAssetParams(ops *OpStream, spec *OpSpec, args []string) error {
 
 type assembleFunc func(*OpStream, *OpSpec, []string) error
 
+// Basic assembly. Any extra bytes of opcode are encoded as byte immediates.
 func asmDefault(ops *OpStream, spec *OpSpec, args []string) error {
 	ops.checkArgs(*spec)
-	if len(spec.Returns) > 0 {
-		ops.tpusha(spec.Returns)
-		ops.trace(" pushes(%s", spec.Returns[0].String())
-		if len(spec.Returns) > 1 {
-			for _, rt := range spec.Returns[1:] {
-				ops.trace(", %s", rt.String())
-			}
-		}
-		ops.trace(")")
```

Comment:
why did this go? meaning both type stack and trace messages

Reply:
It's done in checkArgs() now, so most opcodes don't have to do any extra work.  (Just opcodes that have alternate return types, like field accesses)